docs/guide/protocols/absolutesolvation.rst

```diff
@@ -1,4 +1,94 @@
 Absolute Solvation Protocol
 ===========================
 
-issue #743
\ No newline at end of file
+Overview
+--------
+
+The absolute solvation protocol calculates the free energy change 
+associate with transferring a molecule from vacuum into a solvent.
+
+.. note::
+   Currently, water is the only supported solvent, however, more solvents might be possible in the future.
+
+The absolute hydration free energy is calculated through a thermodynamic cycle. 
+In this cycle, the interactions of the molecule are decoupled, meaning turned off, both in the solvent and in the vacuum phases.
+The absolute hydration free energy is then obtained via summation of free energy differences along the thermodynamic cycle.
+
+.. figure:: img/ahfe_thermocycle.png
+   :scale: 100%
+
+   Thermodynamic cycle for the absolute hydration free energy protocol.
+
+Scientific Details
+------------------
+
+Partial annhilation scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the :class:`.AbsoluteSolvationProtocol` the coulombic interactions of the molecule are fully turned off (annihilated),
+while the Lennard-Jones interactions are decoupled, meaning the intermolecular interactions turned off, keeping the intramolecular Lennard-Jones interactions.
+
+The lambda schedule
+~~~~~~~~~~~~~~~~~~~
+
+Molecular interactions are turned off during an alchemical path using a discrete set of lambda windows. The electrostatic interactions are turned off first, followed by the decoupling of the van-der-Waals interactions. A soft-core potential is applied to the Lennard-Jones potential to avoid instablilites in intermediate lambda windows. 
+
+.. _Simulation details:
+
+Simulation details
+~~~~~~~~~~~~~~~~~~
+
+The protocol applies a LangevinMiddleIntegrator which uses Langevin dynamics, with the LFMiddle discretization [1]_.
+Before running the production MD simulation in the NPT ensemble, the protocol performs a minimization of the system, followed by an equilibration in the NPT ensemble. A MonteCarloBarostat is used in the NPT ensemble to maintain constant pressure.
+
+Getting the free energy estimate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The free energy differences are obtained from simulation data using the MBAR estimator (multistate Bennett acceptance ratio estimator).
```

Comment:
I know it's mentioned below, but it might be worth referencing here that you get the free energy for the entire cycle (unlike the RFE protocol)